source/scripting/World.cs

```diff
@@ -1318,7 +1318,7 @@ public static void ShootBullet(Vector3 sourcePosition, Vector3 targetPosition, P
 		/// <param name="invisible">if set to <c>true</c> explosion is invisible.</param>
 		public static void AddExplosion(Vector3 position, ExplosionType type, float radius, float cameraShake, Ped owner = null, bool aubidble = true, bool invisible = false)
 		{
-			if (Entity.Exists(owner))
+			if (owner.Exists())
```

Comment:
no, now this uses extension 'PoolObject.Exists' method and it checks whether the entity is null before calling 'PoolObject.IsValid', so this won't throw any exceptions.